EvolutionModel/Foodbag.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Evolution.Model {
+    public class Foodbag {
+        private int foodamount;
```

Comment:
We hadden nog iets gezegd toen we met de Aflegstapel bezig waren over het verschil tussen properties en fields. Kijk nog eens naar properties en "getters en setters", of we daar misschien iets nuttigs in zien.

Reply:
Ik begrijp wat je bedoelt, maar niet hoe ik het erin moet zetten. Moet de get set hier toegevoegd worden? Of later bij de functie foodbag?